src/utils/EventUtils.ts

```diff
@@ -86,7 +86,13 @@ export function spreadEvent(args: Result | Record<string, unknown>): { [key: str
   // Truncate all fields which may be bytes32 into a bytes20 string.
   for (const field of knownExtendedAddressFields) {
     if (isDefined(returnedObject[field])) {
-      returnedObject[field] = toAddress(String(returnedObject[field]));
+      let address = String(returnedObject[field]);
+      try {
+        address = toAddress(address);
+      } catch (_e) {
+        _e;
```

Comment:
are we throwing e? logging it?

Reply:
This is just lint appeasement. Added comment about that here: https://github.com/across-protocol/sdk/pull/856/commits/82154e41da9b8c0f3191d3a0ae5e7fb2fcc6cfac